.travis.yml

```diff
@@ -1,24 +1,51 @@
 language: php
 
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+  - nightly
   - hhvm
- 
+
 matrix:
-    allow_failures:
-        - php: hhvm
+  fast_finish: true
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.5.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.6.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=1.3.*
+    - php: 5.6
+      env: FOSUSERBUNDLE_VERSION=2.0.*@dev
```

Comment:
> FOSUserBundle should be tested only against LTS version.

@stloyd why? Symfony 2.5 is [still maintained](http://symfony.com/roadmap?version=2.5#checker) and should be tested. We can remove it when support will be dropped.

> you can use FOSUserBundle 1.3 with Symfony 2.3 and FOSUserBundle 2.0 with Symfony 2.6 for instance

@stof I understand the trick but still think it's quite ugly and messy. Anyway, I will make a patch for it.